src/connection.hpp

```diff
@@ -96,6 +96,12 @@ class Connection
   /// The incoming request.
   std::unique_ptr<Request> request_;
 
+  // string containing http resonse
+  std::string response_string;
+
+  // Buffers for write
```

Comment:
Same for this one. I think it's better to keep the double slashes instead of triple slashes though if that's what we're talking about..